### PR TITLE
[WIP] add postgres operator

### PR DESCRIPTION
**Pull request by linki**
initial files for adding the postgres operator to our setup.

### CODE REVIEW

File: cluster/manifests/postgres-operator/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      application: postgres-operator
+  template:
+    metadata:
+      labels:
+        application: postgres-operator
+        version: ba6529b
+    spec:
+      containers:
+      - name: postgres-operator
+        image: registry.opensource.zalan.do/acid/postgres-operator:ba6529b
```
Comment (hjacobs) [inline cluster/manifests/postgres-operator/deployment.yaml:23]: Can we at least get some 0.1 tag from team ACID? :smirk:

File: cluster/manifests/postgres-operator/config.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+data:
+  service_account_name: operator
+  cluster_labels: application:spilo
+  cluster_name_label: version
+  pod_role_label: spilo-role
+  db_hosted_zone: teapot.zalan.do
+  debug_logging: "true"
+  dns_name_format: '%s.%s.staging.%s'
+  docker_image: registry.opensource.zalan.do/acid/spiloprivate-9.6:1.2-p7
+  etcd_host: etcd-client.default.svc.cluster.local:2379
```
Comment (hjacobs) [inline cluster/manifests/postgres-operator/config.yaml:17]: this looks wrong: the postgres-operator runs in kube-system, but etcd in default namespace?

File: cluster/manifests/postgres-operator/deployment.yaml (hunk 1 above)
Comment (linki) [reply cluster/manifests/postgres-operator/deployment.yaml:23]: That would be great, can we @alexeyklyukin?

File: cluster/manifests/postgres-operator/config.yaml (hunk 2 above)
Comment (linki) [reply cluster/manifests/postgres-operator/config.yaml:17]: Sorry, you're right. This is just a placeholder for now

**Comment by mikkeloscar** [comment]
Closing as the postgres operator is already being deployed via CDP.

**Pull request closed without merging**